src/v2/Apps/Order/OrderApp.tsx

```diff
@@ -183,6 +159,7 @@ class OrderApp extends React.Component<OrderAppProps, {}> {
                     content="width=device-width, initial-scale=1, maximum-scale=5 viewport-fit=cover"
                   />
                 )}
+                {!isEigen && this.renderZendeskScript()}
```

Comment:
This was accidentally removed in a previous PR: https://github.com/artsy/force/pull/7279